app-emulation/qemu/qemu-5.2.0-r2.ebuild

```diff
@@ -1,13 +1,11 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="7"
 
-PYTHON_COMPAT=( python{3_6,3_7} )
+PYTHON_COMPAT=( python3_7 )
```

Comment:
Not sure about the change.

Without `python3_6`, the build fails like:
```
  The following REQUIRED_USE flag constraints are unsatisfied:
    python_targets_python3_7
```

Reply:
I did fix both these issues when fixing the build issues. I'll have a look.